### PR TITLE
Document the simplified release process

### PR DESCRIPTION
**Pull request by mthpvg**
### Planning
https://github.com/upfrontIO/livingdocs-planning/issues/1444

### CODE REVIEW

File: reference-docs/maintenance/release-management/how.md [hunk 1]
```diff
@@ -14,11 +14,8 @@ The release use `v2.1.0` as a base tag.
 #### Optional
 **If** `v2.1.0` is not the latest tag/release and to avoid potential versioning conflicts one need to execute:
 
-1. `git checkout -b release-bump-2.2.0`
-2. `git commit --allow-empty -m "feat: bump minor version to 2.2.0 for release management"`
-3. `git push origin release-bump-2.2.0`
-4. `git branch -D release-bump-2.2.0`
-5. Merge the Pull request in the Github UI and wait for the semantic release script to produce the new `v2.2.0` release and tag.
+1. `./node_modules/@livingdocs/release-tools/li-release push-feat-commit v2.2.0`
```
Comment (peyerluk) [inline reference-docs/maintenance/release-management/how.md]: There is some context missing here. Where is the user supposed to execute this command?

File: reference-docs/maintenance/release-management/how.md [hunk 2]
```diff
@@ -14,11 +14,8 @@ The release use `v2.1.0` as a base tag.
...
+2. Merge the Pull request in the Github UI and wait for the semantic release script to produce the new `v2.2.0` release and tag.
```
Comment (peyerluk) [inline reference-docs/maintenance/release-management/how.md]: Which pull-request? There is no mention of a pull request in this file before this instruction.

**Comment by peyerluk** [comment]
"Simplify how release" is not really a sentence or a title. Maybe something like `Document the simplified release process`?

**Comment by mthpvg** [comment]
@peyerluk , I have addressed your comments in the new commit: https://github.com/upfrontIO/livingdocs/pull/128/commits/ea9bf51e046547c9e2b58e45d92d992e145ba959

This document is also worked on in this PR: https://github.com/upfrontIO/livingdocs/pull/131

File: reference-docs/maintenance/release-management/how.md [hunk 3]
```diff
@@ -7,15 +7,17 @@
 
 ### Assumption
 
-The release use `v2.1.0` as a base tag.
+- All the commands are to be executed at the root of a project. The projects are listed above as upstream applications.
+- The release use `v2.1.0` as a base tag.
```
Comment (peyerluk) [inline reference-docs/maintenance/release-management/how.md]: I think this is confusing as the two repos will have different versions to start with. Plus there is a typo. Do you mean 'The release uses `v2.1.0` as a base tag.' or 'The releases use `v2.1.0` as a base tag.'?

I think there is some missing description what a release is.

E.g.

«A release consists of two release branches in both `livingdocs-server` and `livingdocs-editor`.  Both of those projects use semantic versioning. As an example let's assume we want to create a release-branch on the `livingdocs-server` based on the livingdocs-server@2.1.0 version.»

File: reference-docs/maintenance/release-management/how.md [hunk 4]
```diff
@@ -35,7 +37,8 @@ Livingdocs handles the releases of its two downstreams :
 
 ### Assumption
 
-The latest release of the downstream application is `4.0.0`.
+- All the commands are to be executed at the root of a project. The projects are listed above as downstream Livingdocs applications.
+- The latest release of the downstream application is `4.0.0`.
```
[the remark this answers is not in the document]
Comment (peyerluk) [reply reference-docs/maintenance/release-management/how.md]: And this is the second time the subtitle `Assumption` is used in this document. What is this assumption for? E.g. What are we trying to do in this part of the document?

E.g. What are the scripts `init-release` and `finish-release` doing?

**Comment by peyerluk** [comment]
@mthpvg I think this is outdated with the work @DaRaFF has done. 

Is there something in here that we should salvage? Or should we close this pr?

**Comment by DaRaFF** [comment]
@peyerluk @mthpvg I will close this PR, because I also considered the feedback of this PR in my documentation.

**Pull request closed without merging**